src/app/components/hearing/request-playback-audio/request-playback-audio.component.ts

```diff
@@ -84,6 +88,10 @@ export class RequestPlaybackAudioComponent implements OnChanges {
     return Object.keys(errors).map((errorType) => fieldErrors[fieldName][errorType]);
   }
 
+  public isTranscriber(): boolean {
+    return this.userState.roles.some((x) => x.roleName === 'TRANSCRIBER');
+  }
+
```

Comment:
I think this should belong in the user service?

Reply:
isTranscriber method has now been moved to the User Service